lib/suspenders/app_builder.rb

```diff
@@ -379,6 +379,22 @@ def init_git
       run 'git init'
     end
 
+    def git_init_commit
+      if @@user_choice.present? || @@user_choice.include?(:gitcommit)
+        run 'git add .'
+        run 'git commit -m "Init commit"'
+      end
+    end
+
+    def rvm_gemset_creation_or_ruby_version
+      if system 'rvm -v'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.